lib/active_record/connection_adapters/postgis/oid/spatial.rb

```diff
@@ -25,6 +26,11 @@ def initialize(geo_type: "geometry", srid: 0, has_z: false, has_m: false, geogra
           #   has_z:    false
           #   has_m:    false
           def self.parse_sql_type(sql_type)
+            # Could be nil during type registration
```

Comment:
Not entirely sure why this happens as I'm not fully familiar with active record internals, but this seems to work.